bundle-workflow/scripts/components/OpenSearch-Dashboards/install.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright OpenSearch Contributors.
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+function usage() {
+    echo "Usage: $0 [args]"
+    echo ""
+    echo "Arguments:"
+    echo -e "-a ARTIFACTS\t[Required] Location of build artifacts."
+    echo -e "-o OUTPUT\t[Required] Output path."
+    echo -e "-h help"
+}
+
+while getopts ":h:a:o:" arg; do
+    case $arg in
+        o)
+            OUTPUT=$OPTARG
+            ;;
+        a)
+            ARTIFACTS=$OPTARG
+            ;;
+        h)
+            usage
+            exit 1
+            ;;
+        ?)
+            echo "Invalid option: -${arg}"
+            exit 1
+            ;;
+    esac
+done
+
+mkdir -p "${OUTPUT}/bundle/opensearch-dashboards"
+
+# Untar for the plugins
```

Comment:
You should not be installing plugins here, but let the assemble step take care of it one plugin at a time.